.github/workflows/new_theme_bundle.yml

```diff
@@ -0,0 +1,39 @@
+name: New theme bundle
+
+on:
+  repository_dispatch:
+    types: [new_theme_bundle]
+
+env:
+  release_name: ${{ github.event.client_payload.release }}
+  release_notes_url: ${{ github.event.client_payload.release_notes_url }}
+  bundle_file: ${{ github.event.client_payload.bundle_file }}
+  bundle_url: ${{ github.event.client_payload.bundle_url }}
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setup Perl environment
+        uses: shogo82148/actions-setup-perl@v1.15.3
+
+      - name: Update theme bundle in Antora playbook
+        run: |
+          echo "bundle_url=$bundle_url"
+          perl -007 -i -pe 's?.*ui:.*\n.*bundle:.*\n.*url:.*?ui:\n  bundle:\n    url: '$bundle_url'?' antora-playbook.yml
```

Comment:
perl? Show Time 🥇

Reply:
Yes, I'm not a big fan, but it was thesimplest solution I found ;-)